### PR TITLE
Daedalus and Aegis

### PR DESCRIPTION
**Pull request by Shushishtok**
Fixes https://github.com/ModDota/dota-tutorial/issues/229

- Aegis expiration time increased from 5 minutes to 60 minutes. This is to ensure it doesn't suddenly expire, making a sudden unexpected sound. Unless the player is playing for 60 minutes straight after beating Roshan. But then that's on him.
- Added Daedalus to required items in CH5.

### CODE REVIEW

File: game/scripts/vscripts/Sections/Chapter5/SectionOpening.ts [hunk 1]
```diff
@@ -35,6 +34,7 @@ const requiredState: RequiredState = {
     requireBountyRunes: true,
     requireRoshan: true,
     roshanHitsLikeATruck: true,
+    heroItems: { "item_greater_crit": 1 },
```
Comment (smashthestate) [inline game/scripts/vscripts/Sections/Chapter5/SectionOpening.ts:37]: Can use `heroItems: { [shared.itemDaedalus]: 1 }` instead

File: game/scripts/vscripts/Sections/Chapter5/SectionRoshan.ts [hunk 2]
```diff
@@ -30,7 +30,8 @@ const requiredState: RequiredState = {
         chapter5Blockades.direMidTopRiver,
         chapter5Blockades.midRiverTopSide,
     ],
-    requireRoshan: true
+    requireRoshan: true,
+    heroItems: { "item_greater_crit": 1 },
```
Comment (smashthestate) [inline game/scripts/vscripts/Sections/Chapter5/SectionRoshan.ts:34]: Same as above

File: game/scripts/vscripts/Sections/Chapter5/SectionOpening.ts (hunk 1 above)
Comment (Shushishtok) [reply game/scripts/vscripts/Sections/Chapter5/SectionOpening.ts]: Done

File: game/scripts/vscripts/Sections/Chapter5/SectionRoshan.ts (hunk 2 above)
Comment (Shushishtok) [reply game/scripts/vscripts/Sections/Chapter5/SectionRoshan.ts]: Done

**Pull request merged**